app/views/pages/cookies.md

```diff
@@ -1,126 +1,140 @@
+
+| Name                                                | Purpose                                                                                                                                                        | Expires after                                              |
+| ------                                              | --------                                                                                                                                                       | -------                                                    |
+| `bam-session`                                       | BAM use this cookie to attribute mailing list sign ups back to the agent at a teaching event that presented a QR code to the candidate                         | 30 days                                                    |
+| `_ga`                                               | Google Analytics cookie that helps us count how many people visit our website by tracking if you've visited before                                             | 2 years                                                    |
+| `_gid`                                              | Google Analytics cookie that helps us count how many people visit our website by tracking if you've visited before                                             | 24 hours                                                   |
+| `_gat`                                              | Google Analytics cookie that helps us understand how people use our website                                                                                    | 1 minute                                                   |
+| `AMP_TOKEN`                                         | Uses tokens to help identify your client ID                                                                                                                    | 30 seconds to 1 year                                       |
+| `_ga_76JLPNZBCP` `_ga_2BHBCSRT7L` `_ga_D3DXWHT5D3`  | Used by Google Analytics to find and track an individual session with your device                                                                              | 2 years                                                    |
+| `_gat_UA-179982899-1`                               | Used by Google Analytics to find and track an individual session with your device                                                                              | 1 minute                                                   |
+| `_gaexp`                                            | Used by Google Optimize to identify if you've been part of an experiment or previous experiments which help us test parts of the website                       | Depends on the length of experiment, but typically 90 days |
```

Comment:
Are we still using Google Optimize? is this needed?

Reply:
We need to leave it in for now as it is still on the site - we have a separate ticket to get rid of it